src/OasisCdpManager.t.sol

```diff
@@ -1,19 +1,9 @@
-pragma solidity >= 0.5.0;
+pragma solidity 0.5.12;
```

Comment:
We should upgrade to the latest `dss-deploy` so that this can be `^0.5.12`.

Reply:
yes, let's do this.